ggshield/core/utils.py

```diff
@@ -270,7 +270,7 @@ def load_dot_env() -> None:
                 "GITGUARDIAN_DOTENV_LOCATION does not point to a valid .env file"
             )
 
-    cwd_env = os.path.join("..", ".env")
+    cwd_env = ".env"
```

Comment:
Open question : should we add a more detailed docstring, or do you think the code is self-explanatory ?

Reply:
I think it's self-explanatory, but I wrote it so I am not the right person to ask :)

Going to add something like `# look for a .env in the current directory`.